### PR TITLE
EVAKA-TECH: api-types codegen v2

### PR DESCRIPTION
**Pull request by Joosakur**
#### Summary

- Group api types in different files by first package level to reduce name conflicts
- Improve import config (allows importing Actions correctly)

### CODE REVIEW

File: frontend/src/lib-common/generated/api-types/assistanceaction.d.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    endDate: LocalDate
+    measures: AssistanceMeasure[]
+    otherAction: string
+    startDate: LocalDate
+}
+
+/**
+* Generated from fi.espoo.evaka.assistanceaction.AssistanceMeasure
+*/
+export type AssistanceMeasure = 'SPECIAL_ASSISTANCE_DECISION' | 'INTENSIFIED_ASSISTANCE' | 'EXTENDED_COMPULSORY_EDUCATION' | 'CHILD_SERVICE' | 'CHILD_ACCULTURATION_SUPPORT' | 'TRANSPORT_BENEFIT'
```
Comment (jaahon) [inline frontend/src/lib-common/generated/api-types/assistanceaction.d.ts:46]: I think these string union types would be easier to read if they were formatted like this
```
export type AssistanceMeasure = 
  | 'SPECIAL_ASSISTANCE_DECISION'
  | 'INTENSIFIED_ASSISTANCE'
  ...
```

File: frontend/src/lib-common/generated/api-types/assistanceaction.d.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+/**
+* Generated from fi.espoo.evaka.assistanceaction.AssistanceActionRequest
+*/
+export interface AssistanceActionRequest {
+    actions: string[]
+    endDate: LocalDate
+    measures: AssistanceMeasure[]
+    otherAction: string
+    startDate: LocalDate
+}
```
Comment (jaahon) [inline frontend/src/lib-common/generated/api-types/assistanceaction.d.ts:41]: Minor thing, but I would prefer an indent of 2 spaces to match the rest of our typescript code.

**Review by jaahon — COMMENTED** [review]
Good stuff! 👍 

The `/movieapp` stuff probably doesn't belong here 😄

**Comment by Joosakur** [comment]
> Good stuff! 👍
> 
> The `/movieapp` stuff probably doesn't belong here 😄

argh how did that end up here 😆 some gofore recruitment coding exercise stuff

File: frontend/src/lib-common/generated/api-types/assistanceaction.d.ts (hunk 2 above)
Comment (Joosakur) [reply frontend/src/lib-common/generated/api-types/assistanceaction.d.ts:41]: ah, good point

File: frontend/src/lib-common/generated/api-types/assistanceaction.d.ts (hunk 1 above)
Comment (Joosakur) [reply frontend/src/lib-common/generated/api-types/assistanceaction.d.ts:46]: good idea

**Review by jaahon — APPROVED** [review]

**Pull request merged**